local_server/cosyvoice_server/model_server.py

```
@@ -0,0 +1,167 @@
+    print("---------------------------------------------------------")
+    print("❌ 导入失败！请检查路径是否正确。")
+    print(f"当前尝试加载的路径: {COSYVOICE_PROJECT_ROOT}")
+    print(f"错误信息: {e}")
+    print("请确认该路径下是否有 'cosyvoice' 文件夹。")
+    print("---------------------------------------------------------")
+    sys.exit(1)
+
+# 模型路径 (这里填写具体的模型权重路径，相对于当前 model_server.py 或者是绝对路径)
+# 建议写绝对路径以防万一
+MODEL_DIR = os.path.join(COSYVOICE_PROJECT_ROOT, "pretrained_models/CosyVoice2-0.5B")
+# 或者如果你把模型拷到了 Lanlan 下面：
```

Ruff (0.14.8)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)